config/routes.rb

```diff
@@ -16,13 +16,14 @@
         end
       end
 
-      resources :closures, except: [:show], format: false      
+      resources :closures, except: [:show], format: false
     end
 
     unauthenticated do
       root 'site#index', as: :unauthenticated_root
+      get  '/obrigado', to: 'site#thank_you', as: :thank_you
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.